app/src/main/java/com/sunasterisk/dmealfoodapp/ui/search/SearchFragment.kt

```diff
@@ -2,10 +2,15 @@ package com.sunasterisk.dmealfoodapp.ui.search
 
 import com.sunasterisk.dmealfoodapp.R
 import com.sunasterisk.dmealfoodapp.base.BaseFragment
+import com.sunasterisk.dmealfoodapp.utils.gone
+import com.sunasterisk.dmealfoodapp.utils.show
+import kotlinx.android.synthetic.main.tool_bar_screen_child.*
 
 class SearchFragment : BaseFragment(R.layout.fragment_search) {
 
     override fun onCreatedView() {
+        editTextSearch.show()
+        textTitleScreen.gone()
```

Comment:
```suggestion
        textSearch.show()
        textTitle.gone()
```
edit text nếu để tên dài quá em có thể chỉ để mỗi text cũng đc